raiden/api/validations/channel_validator.py

```diff
@@ -118,3 +118,31 @@ def can_close_channel(token_address,
         )
 
         return channels_to_close
+
+    @staticmethod
+    def validate_and_get_channels_to_settle(token_address,
+                                            partner_addresses,
+                                            registry_address,
+                                            raiden):
```

Comment:
Agree, we can just pass the ChainState here

Reply:
The other cases has raiden too, i just based this method on the other methods on this file, are we going to start changing this now? i mean i can do it but i don't see the advantage since we have other methods on same file with the same structure using raiden as a parameter